servers/rendering/renderer_rd/storage_rd/texture_storage.cpp

```diff
@@ -219,6 +219,34 @@ TextureStorage::TextureStorage() {
 		}
 	}
 
+	{ //create default cubemap
```

Comment:
```suggestion
	{ //create default cubemap array
```
like for `ARRAY_BLACK`, no?

Reply:
Or well, the comments seem backward between the scopes that create `DEFAULT_RD_TEXTURE_CUBEMAP_ARRAY_BLACK` and `DEFAULT_RD_TEXTURE_CUBEMAP_ARRAY_WHITE` (says "default cubemap") and the ones that create `DEFAULT_RD_TEXTURE_CUBEMAP_BLACK` ("default cubemap array") and `DEFAULT_RD_TEXTURE_CUBEMAP_WHITE` ("default cubemap white array").

Might be worth cleaning up to make the comments more accurate (and maybe fix capitalization / punctuation while at it).